lib/taskbook.js

```diff
@@ -235,6 +238,16 @@ class Taskbook {
     return data;
   }
 
+  _filterAfterDate(date = moment().format(DATE_FORMATTER), data = this._data) {
+    Object.entries(data).forEach(([id, item]) => {
```

Comment:
Can this be re-written as the following?

```
Object.entries(data).filter(([id, item]) => {
  return moment(item._timestamp).isAfter(moment(date))
});
```

Reply:
Yes, the `filter` is better than `foreach` for this case.

Reply:
Yes, thanks